packages/server/src/server.ts

```diff
@@ -189,7 +188,7 @@ const logAndHandleResponse = async (methodName: any, methodParams: any, methodFu
 
     const response = await methodFunction(requestId);
     const status = response instanceof JsonRpcError ? response.code.toString() : responseSuccessStatusCode;
-    ms = Date.now() - start;
+    ms = Date.now() - app.getStartTimestamp();
```

Comment:
This is here just for metrics right?
Should we be capturing metrics in `index.ts` instead?
I think the measurements would more accurate since this is the outer logic

Reply:
The index.ts is as minimal as it gets.  The server.ts executes the outer shell.